src/Sarif/Core/Stack.cs

```diff
@@ -120,7 +120,10 @@ public static Stack Create(string stackTrace)
                 {
                     stackFrame.Location = new Location
                     {
-                        FullyQualifiedLogicalName = match.Groups[1].Value
+                        LogicalLocation = new LogicalLocation
+                        {
+                            FullyQualifiedName = match.Groups[1].Value
+                        }
```

Comment:
i should have a null check here

Reply:
why? if it's to avoid and empty logical location in the serialization i can show you how to fix.


---
In reply to: [264979053](https://github.com/Microsoft/sarif-sdk/pull/1347#discussion_r264979053) [](ancestors = 264979053)

Reply:
is the trick to use ShouldSerialize<> thing? ;) 

---
In reply to: [264981118](https://github.com/Microsoft/sarif-sdk/pull/1347#discussion_r264981118) [](ancestors = 264981118,264979053)